apps/frontend/src/pages/CreateClientPage.tsx

```diff
@@ -0,0 +1,71 @@
+import { useRef, useState } from "react";
+import ClientForm from "../components/clients/ClientForm/ClientForm";
+import ContactForm from "../components/clients/ContactForm/ContactForm";
```

Comment:
```suggestion
import ClientForm from "../components/clients/ClientForm/";
import ContactForm from "../components/clients/ContactForm/";
```

Esa era la idea de agregar los index